modules/document_repository/README.md

```diff
@@ -0,0 +1,35 @@
+# Document Repository
+
+## Purpose
+
+ The Document Repository is a tool that provides studies with a centralized location for non-candidate documents. Files may be uploaded and organized under any user-defined category.
+
+## Intended Users
+ 
+ All Loris user and data entry staff could view, update, edit and delete the upload files with right permissions.
+
+## Scope
+
+ It's for all non-candidate documents.
+
+## Permissions
+
+ A user that has "document_repository_view" permission can view and upload files in Document Repository.
+ A user that has "document_repository_delete" permission can delete files in Document Repository.
+
+## Configurations
+
+
+- The Document Repository enables users to upload and organize project files of any type that can easily be viewable for users with appropriate permissions. Give full permissions to store documents on server.
+
+- Enable uploads files:please update the php.ini apache configuration file. 
+```
+session.gc_maxlifetime = 10800  // After this number of seconds, stored data will be seen as 'garbage' and cleaned up by the garbage collection process.
+max_input_time = 10800          // Maximum amount of time each script may spend parsing request data (in seconds)
+max_execution_time = 10800      // Maximum execution time of each script (in seconds)
+upload_max_filesize = 1024M     // Maximum allowed size for uploaded files.
+post_max_size = 1024M           // Maximum size of POST data that PHP will accept.
+```
+
+- A mail server will be required for sending out email notifications about Document Repository updates. 
+
```

Comment:
The documents are uploaded and stored under the modules/document_repository/user_uploads/directory.   Note this directory should be (made?) writable, and care should also be taken to back up these documents / not clobber them when updating code or migrating machines.

Interactions: 
Document repository (notifications?) are visible on the dashboard.